src/rez/bind/python.py

```diff
@@ -23,19 +23,23 @@ def setup_parser(parser):
 
 
 def commands():
-    env.PATH.append('{this.root}/bin')
+    noop = intersects(ephemerals.get_range('dcc_python', '0'), '1')
+    if not noop:
+        env.PATH.append('{this.root}/bin')
 
 
 def post_commands():
-    # these are the builtin modules for this python executable. If we don't
-    # include these, some python behavior can be incorrect.
-    import os
-    import os.path
-
-    path = os.path.join(this.root, "python")  # noqa
-    for dirname in os.listdir(path):
-        path_ = os.path.join(path, dirname)
-        env.PYTHONPATH.append(path_)
+    noop = intersects(ephemerals.get_range('dcc_python', '0'), '1')
+    if not noop:
+        # these are the builtin modules for this python executable. If we don't
+        # include these, some python behavior can be incorrect.
+        import os
+        import os.path
```

Comment:
Do we still need these imports here?

Reply:
yeah, it's still needed. this function gets written out into the python package.py file. although the if not noop should go after the imports. i'll fix that